addons/miscMedical/ACE_Medical_Treatment_Actions.hpp

```diff
@@ -7,6 +7,8 @@ class ace_medical_treatment_actions {
         items[] = {"potato_naloxone"};
         litter[] = {{"ACE_MedicalLitter_atropine"}};
     };
+    class Painkillers: Morphine {};
+    class Pkblister: Painkillers {};
```

Comment:
if we are assuming a blisterpack painkiller is the same as a painkiller, we dont need this